Jenkinsfile

```diff
@@ -41,8 +41,11 @@ properties([
         stringParam(defaultValue: '10.40.9.2', description: 'IP of the WAN board',
             name: 'WAN_IP'),
         stringParam(defaultValue: '/home/jenkins/boardfarm/boardfarm',
-            description: 'Path to the a local copy of Boardfarm repository',
-            name: 'BOARDFARM_DIRECTORY'),
+            description: 'Path to a local copy of OTA scripts',
+            name: 'OTA_DIRECTORY'),
+        stringParam(defaultValue: 'https://github.com/CreatorDev/boardfarm.git',
```

Comment:
I would have expected you to be using the Jenkins scm steps so that it can keep track of versions? See the openwrt Jenkinsfile for reference